redbot/cogs/modlog/modlog.py

```diff
@@ -143,8 +149,18 @@ async def casesfor(self, ctx: commands.Context, *, member: Union[discord.Member,
             return await ctx.send(_("That user does not have any cases."))
 
         embed_requested = await ctx.embed_requested()
-
-        rendered_cases = [await case.message_content(embed=embed_requested) for case in cases]
+        if embed_requested:
+            rendered_cases = [await case.message_content(embed=True) for case in cases]
+        elif not embed_requested:
+            rendered_cases = []
+            for case in cases:
+                message = _("{case}\n**Timestamp**: {timestamp}").format(
+                    case=await case.message_content(embed=False),
+                    timestamp=datetime.fromtimestamp(case.created_at).strftime(
+                        "%Y-%m-%d %H:%M:%S"
```

Comment:
Updated handling per #4017:
```suggestion
                    timestamp=datetime.utcfromtimestamp(case.created_at).strftime(
                        "%Y-%m-%d %H:%M:%S UTC"
```